libuavcan_drivers/linux/scripts/uavcan_add_vcan

```diff
@@ -19,11 +19,14 @@ if [ "$1" == '--help' ] || [ "$1" == '-h' ]; then echo "$HELP"; exit; fi
 
 IFACE="$1"
 
-if [ $(ifconfig -a | grep -c "^$IFACE") -eq "1" ]; then
-    ifconfig $IFACE up
+ip link show $IFACE > /dev/null
+if [ $? == 0 ]; then
+    ip link set up $IFACE
     exit
 fi
 
+echo  "load driver and start $IFACE" 
```

Comment:
Not sure what it means, could you elaborate, please? Why can't we do this automatically? Everything that can be sensibly automated, should be.